### PR TITLE
minor tidying

### PR DESCRIPTION
**Pull request by blentz**
Some minor cleanups

 - fixing ReportMap's public properties.
- drop QueryHandler's pass-through `execute_query()` method, renamed `execute_sum_query()` to `execute_query()`.

### CODE REVIEW

File: koku/api/report/aws/aws_query_handler.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def _set_tag_filters(self, filters):
             filters.add(QueryFilter(operation='tags__iexact', parameter='{}'))
         return super()._set_tag_filters(filters)
 
-    def execute_sum_query(self):
+    def execute_query(self):
         """Execute query and return provided data when self.is_sum == True.
```
Comment (dccurtis) [inline koku/api/report/aws/aws_query_handler.py]: While we're at it, can you update this comment?

File: koku/api/report/ocp_aws/ocp_aws_query_handler.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def __init__(self, query_parameters, url_data,
             self._mapper = ProviderMap(provider=provider,
                                        report_type=self._report_type)
 
-    def execute_sum_query(self):
+    def execute_query(self):
         """Execute query and return provided data when self.is_sum == True.
```
Comment (dccurtis) [inline koku/api/report/ocp_aws/ocp_aws_query_handler.py]: Same here

File: koku/api/report/aws/aws_query_handler.py (hunk 1 above)
Comment (dccurtis) [inline koku/api/report/aws/aws_query_handler.py]:
```suggestion
        """Execute query and return provided data.
```

File: koku/api/report/ocp_aws/ocp_aws_query_handler.py (hunk 2 above)
Comment (dccurtis) [inline koku/api/report/ocp_aws/ocp_aws_query_handler.py]:
```suggestion
        """Execute query and return provided data.
```

File: koku/api/report/ocp/ocp_query_handler.py [hunk 3]
```diff
@@ -155,21 +155,12 @@ def execute_sum_query(self):
...
-        Returns:
-            (Dict): Dictionary response of query params, data, and total
-
-        """
-        return self.execute_sum_query()
-
     def get_rank_window_function(self, group_by_value):
         """Generate a limit ranking window function."""
-        tag_column = self._mapper._provider_map.get('tag_column')
+        tag_column = self._mapper.provider_map.get('tag_column')
```
Comment (adberglund) [inline koku/api/report/ocp/ocp_query_handler.py]:
```suggestion
        tag_column = self._mapper.tag_column
```

**Pull request merged**